modules/ROOT/pages/installing-an-enterprise-license.adoc

```diff
@@ -40,15 +40,13 @@ Sometimes the license installation fails and it might be necessary to manually d
 
 == Download your License Key File
 
-* Log in to https://help.mulesoft.com[the Support portal]. If you do not have credentials to log in, please contact your Customer Success Manager.
+. Log in to https://help.mulesoft.com[the Support portal^] using your login information. If you do not have credentials to log in, please contact your Customer Success Manager.
+. Click on the *Support* tab located on the navigation menu of the MuleSoft Help Center home page.
+. Click on the *Subscriptions* tab located halfway through the page.
+. Click on the *Subscription Name* of the subscription you would like a license key for. Please note that you must click on the Subscription Name (second column on the right) or you will not be forwarded to the correct page.
+. Click on the *License ID* number located at the end of the page on the right.
+. Click on the *License Key* file located in the Notes & Attachments section at the beginning of the page on the right to download your license key.
```

Comment:
I'm not sure about the meaning of "section at the beginning of the page on the right". Do you mean "at the top right of the page"? Is it necessary to give the location?

Reply:
Also, should Notes & Attachments be bolded?